tools/c7n_azure/tests_azure/tests_resources/test_search.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2019 Microsoft Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from ..azure_common import BaseTest, arm_template
+
+
+class SearchTest(BaseTest):
+    def setUp(self):
+        super(SearchTest, self).setUp()
+
+    def test_search_schema_validate(self):
```

Comment:
we shouldn't be doing these tests, all policies validate in ci, having a separate one is extraneous.